packages/atomic/cypress/integration/result-list/result-components-v1/result-number.cypress.ts

```diff
@@ -0,0 +1,259 @@
+import {generateComponentHTML} from '../../../fixtures/test-fixture';
+import {
+  interceptSearchResponse,
+  setUpPage,
+} from '../../../utils/setupComponent';
+import {
+  executeFirstSearch,
+  setLanguage,
+} from '../../search-interface-utils.cypress';
+import {
+  generateResultList,
+  generateResultTemplate,
+  getFirstResult,
+} from '../result-list-v1-selectors';
+import {ResultNumberSelectors} from './result-number-selectors';
+
+describe('Result Number Component', () => {
+  function setupResultNumberPage(
+    props: Record<string, string | number>,
+    executeSearch = true,
+    slot = ''
+  ) {
+    const component = generateComponentHTML(
+      ResultNumberSelectors.component,
+      props
+    );
+    component.innerHTML = slot;
+    setUpPage(
+      generateResultList(
+        generateResultTemplate({
+          bottomMetadata: component.outerHTML,
+        })
+      ),
+      executeSearch
+    );
+  }
+
+  function getFirstResultNumber() {
+    return getFirstResult().find(ResultNumberSelectors.component);
+  }
+
+  describe('when not used inside a result template', () => {
+    beforeEach(() => {
+      setUpPage(
+        generateComponentHTML(ResultNumberSelectors.component).outerHTML
+      );
+    });
+
+    it.skip('should remove the component from the DOM', () => {
+      cy.get(ResultNumberSelectors.component).should('not.exist');
+    });
+
+    it('should log a console error', () => {
+      cy.get(ResultNumberSelectors.component)
+        .find('atomic-component-error')
+        .should('exist');
+    });
+  });
+
+  describe('when the field does not exist for the result', () => {
+    beforeEach(() => {
+      setupResultNumberPage({
+        field: 'thisfielddoesnotexist',
+      });
+    });
+
+    it.skip('should remove the component from the DOM', () => {
+      getFirstResultNumber().should('not.exist');
+    });
+  });
+
+  describe('when the field value cannot be parsed to a number', () => {
+    beforeEach(() => {
+      const field = 'my-field';
+      setupResultNumberPage(
+        {
+          field,
+        },
+        false
+      );
+      interceptSearchResponse((response) =>
+        response.results.forEach((result) => (result.raw[field] = 'Abc'))
+      );
+      executeFirstSearch();
+    });
+
+    it.skip('should remove the component from the DOM', () => {
+      getFirstResultNumber().should('not.exist');
+    });
```

Comment:
This test doesn't pass yet but I'm not sure if we should show an error or remove the component when the indexed value isn't a number.

What do you guys think?